uber/templates/dealer_admin/form.html

```diff
@@ -32,7 +32,7 @@
         }
     }
 
-    $(function() {
```

Comment:
Whoops, sometimes I hit the ` key instead of Esc. I'll fix this in the next commit!